controllers/control/cassandratask_controller.go

```diff
@@ -451,7 +502,16 @@ func (r *CassandraTaskReconciler) reconcileEveryPodTask(ctx context.Context, dc
 	}
 
 	failed, completed := 0, 0
+
 	for idx, pod := range dcPods {
+		// TODO Do we need post-pod processing functionality also? In case we need to wait for some other event to happen (processed by cass-operator).
+		//		or could we requeue with the filter instead of only bypassing? Or could we requeue in the Async / SyncFunc?
```

Comment:
Currently, since we don't have post-processing (that would wait for resources to get back), the task could actually restart or replace all the nodes in the Datacenter before the Datacenter can rebootstrap any of the nodes (thus, taking the cluster down).